mobile/src/test/java/com/google/samples/apps/iosched/ui/schedule/ScheduleViewModelTest.kt

```diff
@@ -382,7 +382,7 @@ class ScheduleViewModelTest {
             viewModel.scrollToStartOfDay(TestData.TestConferenceDays[0])
 
             val result = mutableListOf<ScheduleScrollEvent>()
-            val job = launch {
+            val job = launch(UnconfinedTestDispatcher()) {
```

Comment:
Nit: Why not using the dispatcher from the coroutine rule?

Reply:
It doesn't really matter whether you reuse the rule's dispatcher or create a new one, if both are of the same type (`UnconfinedTestDispatcher`, in this case). This way it's more obvious why a dispatcher is being passed in to `launch` - to have the coroutine run eagerly.